<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>香港</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>武汉</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_led">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>turn on led</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_spk">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>turn on spk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_fan">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>turn on fan</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_fan">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_fan">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_spk">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_spk">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_led">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_led">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>700</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The copyright of this client software is owned by Peter Chen personally. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
